src/Slider.jsx

```diff
@@ -29,7 +29,7 @@ function killEvent(ev) {
 
 class Button extends React.Component {
   render() {
-    return <button {...this.props} type="button" />;
+    return <button {...this.props} aria-label="Slider Handle" type="button" />;
```

Comment:
I thought about if we should let the consumers pass this value but decided against it. 

* This is the default case.
* If you want to customize your slider handles you can pass in your own custom component.

But should we make the label say something like `Slider Handle N`?